cypress/integration/testBorna/test.cy.js

```diff
@@ -0,0 +1,179 @@
+        )
+            .eq(0)
+            .click();
+
+        cy.contains("Your export is now ready!").should("exist");
+        cy.wait(1000);
+    });
+
+    it("Test the functionality of the edit block button", () => {
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.contains(block_category_name).click();
+        cy.get(
+            ".rmwc-icon.rmwc-icon--component.material-icons.css-1fdpwmn-EditButton.eglpe7i5.mdc-icon-button"
+        )
+            .eq(0)
+            .click();
+        //cy.get("css-mt1ik6-StyledDivider.eeco1sf0").click().type("Testing Editing Block"); Does not work
+        cy.contains("Save Changes").click();
+
+        getPlaceholder("Search blocks").should("exist");
+        cy.wait(1000);
+    });
+
+    it("Test the exportation of all blocks", () => {
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.findByTestId("pb-blocks-list-options-menu").click();
+        cy.contains("Export all blocks").click();
+
+        cy.contains("Your export is now ready!").should("exist");
+        cy.wait(1000);
+    });
+
+    it("Test the exportation of all blocks from the current category", () => {
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.contains(block_category_name).click();
+
+        cy.findByTestId("pb-blocks-list-options-menu").click();
+        cy.contains("Export blocks from current category").click();
+
+        cy.contains("Your export is now ready!").should("exist");
+    });
+
+    it("Test deletion of the 2 previously created page builder block modules", () => {
+        cy.visit("/page-builder/page-blocks");
+        getPlaceholder("Search blocks").should("exist");
+        cy.contains(block_category_name).click();
+
+        cy.get(
+            ".rmwc-icon.rmwc-icon--component.material-icons.css-1cypnkk-DeleteButton.eglpe7i4.mdc-icon-button"
+        )
+            .eq(0)
+            .click();
+        cy.findByTestId("confirmationdialog-confirm-action").click();
+        cy.wait(2000);
+        cy.get(
+            ".rmwc-icon.rmwc-icon--component.material-icons.css-1cypnkk-DeleteButton.eglpe7i4.mdc-icon-button"
+        )
+            .eq(0)
+            .click();
+        cy.findByTestId("confirmationdialog-confirm-action").click();
+        cy.wait(2000);
+
+        cy.contains("New Block").should("exist");
+    });
+
+    it("Test deletion of the previously created page builder category", () => {
+        cy.visit("/page-builder/block-categories");
+        getPlaceholder("Search block categories").should("exist");
+        cy.wait(100);
+        cy.get(".rmwc-icon.rmwc-icon--component.material-icons.mdc-icon-button")
+            .eq(0)
+            .click({ force: true }); //cannot get the button
```

Comment:
Is this comment relevant here?

BTW, a quick heads up... We're a bit nitpicky when it comes to writing comments. We usually reject comments that include grammar mistakes or are confusing. This is mainly because our code is being often reviewed by larger companies, and we want to leave the best possible impression on them.

Reply:
Fully understandable, the comment was referring to the fact that I cannot reach this button as it requires me to hover over the specified element to make it visible. I was hoping somebody would already know how to get to this button. If not, I would personally look in the hover possibilities.